CMakeLists.txt

```diff
@@ -107,12 +107,15 @@ endif()
 
 
 option(BREAKPAD_HANDLER "Build with breakpad exception handler " OFF)
+message(BREAKPAD_HANDLER - ${BREAKPAD_HANDLER})
```

Comment:
Is this required? Or is just a debug message?

Reply:
Just debug